drivers/serial/uart_mcux_lpuart.c

```diff
@@ -616,37 +616,22 @@ static int uart_mcux_lpuart_dma_replace_rx_buffer(const struct device *dev)
 	struct mcux_lpuart_data *data = (struct mcux_lpuart_data *)dev->data;
 	const struct mcux_lpuart_config *config = dev->config;
 	LPUART_Type *lpuart = config->base;
-	struct mcux_lpuart_rx_dma_params *rx_dma_params = &data->async.rx_dma_params;
+	// struct mcux_lpuart_rx_dma_params *rx_dma_params = &data->async.rx_dma_params;
```

Comment:
why remove this, yo actually change the rx_dma_params

Reply:
I did not change it.  next_rx_buffer is a member of data->async